tests/CppUTestExt/MockFailureTest.cpp

```diff
@@ -205,9 +205,9 @@ TEST(MockFailureTest, MockExpectedParameterDidntHappenFailure)
     MockExpectedParameterDidntHappenFailure failure(UtestShell::getCurrent(), "foo", *list);
     STRCMP_EQUAL("Mock Failure: Expected parameter for function \"foo\" did not happen.\n"
                  "\tEXPECTED calls that DID NOT happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <2>, const char* boo: <str>\n"
+                 "\t\tfoo -> int bar: <2 (0x00000002)>, const char* boo: <str>\n"
                  "\tACTUAL calls that DID happen related to function: foo\n"
-                 "\t\tfoo -> int bar: <10>, const char* boo: <bleh>\n"
+                 "\t\tfoo -> int bar: <10 (0x0000000a)>, const char* boo: <bleh>\n"
```

Comment:
Then there is another concern that, should all the formate follow the (0x0000000a)  or (0xa)?

Seems we get into endless option selecting